substrate/frame/identity/src/migration.rs

```diff
@@ -124,3 +147,482 @@ pub mod v1 {
 		}
 	}
 }
+
+pub mod v2 {
+	use super::*;
+	use frame_support::{
+		migrations::{MigrationId, SteppedMigration, SteppedMigrationError},
+		storage_alias,
+		weights::WeightMeter,
+	};
+
+	type HashedKey = BoundedVec<u8, ConstU32<256>>;
+	// The resulting state of the step and the actual weight consumed.
+	type StepResultOf<T> =
+		(MigrationState<<T as frame_system::Config>::AccountId, Username<T>, Suffix<T>>, Weight);
+
+	mod v1 {
```

Comment:
I think the storage definitions could live outside like `storage::{v0, v1, v2}` or something so that all migrations can use them..  
We dont really have a convention for this.

Reply:
Refactored outside the migration mods.